commcare_connect/templates/opportunity/opportunity_list.html

```diff
@@ -15,19 +72,23 @@ <h1> Opportunities
         </span>
       </h1>
     </div>
-    <div class="pb-4">
+    <div class="pb-4" x-data="sortableTable()">
```

Comment:
This looks like it will only sort the items displayed, but for paginated data, you won't get the true first X items. I think this might need to be implemented like the paginations controls, to rerender the table, and fetch the data from the backend. That likely makes the final implementation simpler (since you won't need much js, just click functions that make ajax calls). Let me know if I am misunderstanding how this works.

Reply:
> This looks like it will only sort the items displayed, but for paginated data, you won't get the true first X items.

Yes, that's right. This sorts client side data only. The server side sorting could be done, but it's entirely different approach and does need much rework. My understanding of the ticket was that they were looking for client side sorting.